kubernetes/uptime-kuma/deployment.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: uptime-kuma
+  name: uptime-kuma
+spec:
+  selector:
+    matchLabels:
+      component: uptime-kuma
+  replicas: 3
```

Comment:
Don't think multiple instances will work. Bakend uses sqlite, and each instance would perform their checks independently.
Also, your deployment uses a PVC marked as `ReadWriteOnce` which won't work.

Reply:
Yes, socket server is not happy with replicas.